### DISCUSSION TITLE
Does pymc use Numba CPU parallelization options?

### DISCUSSION THREAD

**Discussion by diptorupd** [pymc-devs/pytensor#476] [Q&A]
I am a developer of the [numba-dpex](https://github.com/IntelPython/numba-dpex) project that is extending Numba for Intel devices. 

I want to understand how `pymc` currently uses Numba (if at all) and if there are already places where Numba's auto-parallelization is used or can be used. My goal is to explore use cases that will help drive numba-dpex's design and better understand how people use Numba auto-parallelization.

**Comment by twiecki** [comment]
While PyMC does not use numba, PyMC's tensor library -- PyTensor -- does. So you can compile PyMC models to numba, but you won't find any reference of numba in the pymc code base. A good place to start checking it out in PyTensor might be this: https://github.com/search?q=repo%3Apymc-devs%2Fpytensor%20numba&type=code
Certainly feel free to ask any questions, but the pytensor discussion forum might be a better place.
